packages/client/src/flow/addSigner.tx.js

```diff
@@ -1,18 +1,31 @@
 export const ADD_SIGNER = `
-import DAOTreasury from 0xDAOTreasury
-import TreasuryActions from 0xTreasuryActions
+  import DAOTreasury from 0xDAOTreasury
+  import TreasuryActions from 0xTreasuryActions
+  import MyMultiSig from 0xMyMultiSig
 
-  transaction(additionalSigner: Address) {
+  transaction(additionalSigner: Address, message: String, keyIds: [UInt64], signatures: [String], signatureBlock: UInt64) {
     
-    let Treasury: &DAOTreasury.Treasury{DAOTreasury.TreasuryPublic}
-
+    let treasury: &DAOTreasury.Treasury{DAOTreasury.TreasuryPublic}
+    let action: AnyStruct{MyMultiSig.Action}
+    let messageSignaturePayload: MyMultiSig.MessageSignaturePayload
+  
     prepare(signer: AuthAccount) {
-      self.Treasury = signer.borrow<&DAOTreasury.Treasury>(from: DAOTreasury.TreasuryStoragePath)
+      self.treasury = signer.borrow<&DAOTreasury.Treasury>(from: DAOTreasury.TreasuryStoragePath)
```

Comment:
We should add the treasury Address to the args of this transaction.  This `signer.borrow` will only work if the Treasury resource is stored in the signer's storage.